### PR TITLE
Updated make script + bug fix

### PR DESCRIPTION
**Pull request by gulrotkake**
Hi,

after a few minor modifications (and compiling with the latest version of emscripten) I was able to get toFastBitmap to work. Here's a code snippet from a local test:

``` javascript
{
    var reader = new FileReader();
    reader.onload = function(e) {
        console.log('Converting to image, please wait');
        var img = new Image();
        var rawson = new Rawson.File(file.name, e.target.result);

        img.src = rawson.toFastBitmap();//rawson.extractThumbnail();

        img.onload = function() {
            console.log(img.width+'x'+img.height);
            cb(null, img);
        };
    };
    console.log('Loading... Please wait.');
    reader.readAsBinaryString(file);
}
```

### CODE REVIEW

File: lib/rawson.js [hunk 1]
```diff
@@ -97,7 +97,7 @@ Rawson.File.prototype.toFastBitmap = function (ppmdata){
     dcraw.run(['-h','/' + this.filename]);
     var ppmName = this.filename.split('.')[0] + '.ppm';
     if (dcraw.FS.root.contents[ppmName]){
-        this._PNMtoIMG(this.FS.root.contents[ppmName]);
+        return this._PNMtoIMG(this.FS.root.contents[ppmName]);
```
Comment (fbuchinger) [inline lib/rawson.js]: shouldn't this be this.FS.root.contents[ppmName].contents?

**Comment by fbuchinger** [comment]
Hi Michael,

really cool that you are working on my almost hibernated rawson.js port ! :)  I'm glad that you got the `toFastBitmap` method working - i just found a little error referencing the contents of the generated .ppm file (see my comment). Unfortunately `toFastBitmap` doesn't show a real benefit over `extractThumbnail`for my Olympus ORF raw files (computation time is 10 times larger, but the output image is not that much bigger than the embedded thumbnail - the thumbnail is 1600x1200, `toFastBitmap` gives me 1860x1400 pixels). Of course this situation might be different for other raw formats.

Would you mind to check if `toBitmap` works in the lowest quality setting ?('-q 0' option in dcraw) - this would allow us to use the full-scale raw image.

thanks,

Franz

File: lib/rawson.js (hunk 1 above)
Comment (gulrotkake) [inline lib/rawson.js]: Yes, you're absolutely right, it should be .contents, that's what I get for copying, then forking, pasting back and sending a pull request :)

**Comment by gulrotkake** [comment]
toBitmap seems to work fine, haven't tried putting it on a canvas (or in an image) yet, but the following code:

``` javascript
        console.log('Converting to image, please wait');
        console.time('toBitmap');
        var raw = rawson.toBitmap();
        console.timeEnd('toBitmap');
```

gave the following result:

```
Converting to image, please wait
toBitmap: 32561.014ms 
```

where the returned object was a lovely array of length 12632081, as seen using Chrome's inspector:

```
Object {read: 256, write: 128, timestamp: 1360609488460, inodeNumber: 11, isDevice: false…}
    contents: Array[12632081]
    inodeNumber: 11
    isDevice: false
    isFolder: false
    read: 256
    timestamp: 1360609488460
    write: 128
```

for reference, the dcraw arguments:

```
dcraw.run(['-D','-q', '0', '/' + this.filename]);
```

**Pull request merged**